tests/QA/test_param.py

```diff
@@ -186,6 +186,80 @@ def state_cb(name, state):
             with pytest.raises(AttributeError):
                 scf.cf.param.persistent_get_state(param, state_cb)
 
+    def test_param_persistent_stress(self, test_setup):
+        def get_persistent_state(cf, complete_param_name):
+            wait_for_callback_event = Event()
+
+            def state_callback(complete_name, state):
+                wait_for_callback_event.set()
+
+            cf.param.persistent_get_state(complete_param_name, state_callback)
+            wait_for_callback_event.wait()
+
+        def persist_parameter(cf, complete_param_name):
+            wait_for_callback_event = Event()
+
+            def is_stored_callback(complete_name, success):
+                wait_for_callback_event.set()
+
+            cf.param.persistent_store(complete_param_name, callback=is_stored_callback)
+            wait_for_callback_event.wait()
```

Comment:
All these wait will wait forever if something goes wrong and block tests running in the Crazylab. Perhaps you could have a timeout and an assert?

```
assert wait_for_callback_event.wait(timeout=5)
```
Since `wait()` returns `False` on timeout.